### DISCUSSION TITLE
Allow user to place issues in classic projects even after making a new project

### DISCUSSION THREAD

**Discussion by brennansingular** [community/community#32161] [Mobile]
If you start to use the latest version of projects in your organization, your phone no longer allows you to place any issues in the classic projects you have open in the repo

The only options showing now are the new version of projects

This has seriously crippled my workflow as using the mobile version and iPad version is very useful to me but is causing a lot of mayhem for me because I use projects to set up to do tasks

Now I have to go on the computer after creating issues on my device and that is making it harder to not let things slip through the cracks

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.78.0 (76820214)
Device: iPhone 12 Pro (iOS 15.6.1)
</details>

**Comment by HenSquared** [comment]
Hi @brennansingular,

Thanks for the feedback! I'm sorry to hear this has messed up your workflow. 

I think there may be some unintended behavior here. I've opened up an issue to track this and will update you when I find some more info!

Thanks :)

**Reply by brennansingular** [reply]
Thank you!

**Comment by HenSquared** [comment]
You may also find these docs useful if you are interested in migrating a classic project to the new Projects experience. 
https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/migrating-from-projects-classic

**Reply by brennansingular** [reply]
Thanks again, will look into this... might as well
